src/runtime_src/CMakeLists.txt

```diff
@@ -22,6 +22,7 @@ if(XRT_VERBOSE)
 endif()
 
 add_compile_options("-fPIC")
+add_compile_options("-fconcepts")
```

Comment:
Remove please.

Reply:
removed.  It was needed because of auto in  closeApplicationOnMagicStrFound(const auto &matchString). Now I have modified closeApplicationOnMagicStrFound accordingly.